scripts/scripts.js

```diff
@@ -390,6 +390,7 @@ function pushPageLoadToDataLayer() {
  * @param {Element} doc The container element
  */
 async function loadEager(doc) {
+  createMetadata('referrer', 'no-referrer-when-downgrade');
```

Comment:
Since this is static. Should we rather move it to `head.html`?

Reply:
code moved to head.html